api_tests/users/views/test_user_settings_detail.py

```diff
@@ -84,12 +84,6 @@ def test_update_two_factor_permissions(self, app, user_one, user_two, url, paylo
         assert res.status_code == 403
 
     def test_update_two_factor_enabled(self, app, user_one, url, payload):
-        # Invalid data type
-        payload['data']['attributes']['two_factor_enabled'] = 'Yes'
-        res = app.patch_json_api(url, payload, auth=user_one.auth, expect_errors=True)
-        assert res.status_code == 400
-        assert res.json['errors'][0]['detail'] == '"Yes" is not a valid boolean.'
```

Comment:
As discussed, this is removed due to 'Yes' is now a valid boolean. We probably need a new invalid case (e.g. `'yEs'`) instead of removing this test.